core-bundle/src/Resources/contao/pages/PageRegular.php

```diff
@@ -481,6 +481,13 @@ protected function createTemplate($objPage, $objLayout)
 			$this->Template->positions = $arrPositions;
 		}
 
+		// Add the check cookies url to the fe_page template only if the alwaysLoadFromCache
+		// checkbox is configured
+		$this->Template->checkCookiesUrl = '';
+		if ($objPage->alwaysLoadFromCache) {
+			$this->Template->checkCookiesUrl = System::getContainer()->get('router')->generate('contao_frontend_check_cookies');
```

Comment:
Instead of creating an actual route that should not be used because its *internal*, could we use a controller reference throught `_fragments` and sign the request?

Reply:
I don't think we should do that. This is a public route so it doesn't need signing. If you want to call that route, feel free to do so, who cares. It's not internal at all. Or did you mean "internal" because nobody else should be using it? Well then, so be it. If people use it then they do...you're not supposed to use anything marked with `@internal` either but you still can. I think it's correct the way it is.

Reply:
it just also "pollutes" the routing 🤷‍♂